### ISSUE TITLE
Book page: Book title link lacks information for usage 

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#188]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

On a book page, the book title is a link that goes back to the book's cover page. For visual and non-visual users it's not immediately clear where the link will go.

Possible solution:
Use the `title` attribute to clarify the purpose of the link.

```
<a href="https://mobydick.pressbooks.com/" title="Go to cover page of Moby Dick" rel="home">Moby Dick</a>
```
